packages/node_modules/samples/browser-call-with-screenshare/index.html

```diff
@@ -44,16 +44,46 @@
       <main>
         <h1>Local Screensharing</h1>
         <p>This sample demonstrates sharing your local screen</p>
+
+        <form id="typeCredentials">
+          <fieldset>
+            <legend>Pick Method of Getting Access Token</legend>
+              <div>
+                <input type="radio" id="oauth" name="drone" onchange="handleRadio()" value="oauth" checked>
+                <label for="huey">OAuth</label>
+              </div>
+              <div>
+                <input type="radio" id="paste" name="drone" onchange="handleRadio()" value="paste">
+                <label for="huey">Paste Access Token</label>
+              </div>
+            </div>
+          </fieldset>
+        </form>
+
         <form id="credentials">
+          <fieldset id="oauthForm" style="display: block;">
```

Comment:
```suggestion
          <fieldset id="oauth" style="display: block;">
```

Reply:
I decided to call the id as oauthForm because I already named the radio's id as oauth.